### PR TITLE
Added documentation about prometheus collection integration and how t…

### PR DESCRIPTION
**Pull request by acalhounRH**
Added documentation about Prometheus collection integration and how to extend functionality to other workloads

@dry923 
@bengland2 
@ekuric 
@jtaleric 
@rsevilla87

### CODE REVIEW

**Comment by bengland2** [comment]
@acalhounRH , thank you, this is much better.   One thing I don't understand though - why in the prometheus: section do you have to specify es_server: and es_port: when it is already specified under elasticsearch: above?   Seems like you wouldn't use a different ES server for prom data and test results would you?  At least the default should be the same ES server used in the elasticsearch: section?   Anything that reduces number of input parameters needed for the average user significantly increases the chance that they will succeed at using it.  -dumb ole ben

**Comment by acalhounRH** [comment]
Thank you Ben for the input, lets submit a issues for defaulting the prom ES instance if the es_server is not present.

File: docs/elastic.md [hunk 1]
```diff
@@ -50,3 +88,311 @@ spec:
...
+      threads: 1
+      file_size: 0
+      files: 100000
+```
+   
+#### Retrieving Openshift Prometheus info
+
+In order to query Prometheus we must provide the URL and access token. 
+
+The URL can be captured by executing `oc -n openshift-monitoring get routes | grep prometheus`.
```
Comment (rsevilla87) [inline docs/elastic.md:148]: It's also possible (and recommended), to use the internal prometheus service addres, which is always the same for every cluster and prevents traffic to go out and in the cluster. (https://prometheus-k8s.openshift-monitoring.svc.cluster.local:9091)

File: docs/elastic.md [hunk 2]
```diff
@@ -50,3 +88,311 @@ spec:
...
+
+Command to setup a service account and retrieve token: 
+
+```
+oc create serviceaccount snafu -n my-ripsaw
+oc create clusterrolebinding grafana-cluster-monitoring-view --clusterrole=cluster-monitoring-view --serviceaccount=my-ripsaw:snafu
+oc sa get-token snafu -n my-ripsaw
+```
+
+*NOTE:* Service account tokens can expire, for long running test it is recommend to either set the necessary permissions
```
Comment (rsevilla87) [inline docs/elastic.md:161]: By default service account tokens doesn't expire. Rather user tokens expire. i.e (oc whoami -t)

**Comment by acalhounRH** [comment]
Updated and corrected doc's based on @rsevilla87 comments. 

Ready to merge.

**Review by rsevilla87 — APPROVED** [review]
This is great @acalhounRH . LGTM!

**Comment by rsevilla87** [comment]
> @acalhounRH , thank you, this is much better. One thing I don't understand though - why in the prometheus: section do you have to specify es_server: and es_port: when it is already specified under elasticsearch: above? Seems like you wouldn't use a different ES server for prom data and test results would you? At least the default should be the same ES server used in the elasticsearch: section? Anything that reduces number of input parameters needed for the average user significantly increases the chance that they will succeed at using it. -dumb ole ben

FYI: There's an ongoing effort to get rid of ES port variables at https://github.com/cloud-bulldozer/benchmark-operator/pull/438

**Pull request merged**